pkg/routeagent/controllers/route/route.go

```diff
@@ -260,6 +264,40 @@ func (r *Controller) runPodWorker() {
 	}
 }
 
+func (r *Controller) overlappingSubnets(remoteSubnets []string) bool {
```

Comment:
This looks like an operation that would get costly with scale, so hoping this code is only hit when we're coming up.

Reply:
This will be validated when the Submariner route agent is coming up and also at runtime when a new cluster is added.